scripts/build_jekyll_maven.sh

```diff
@@ -45,8 +45,21 @@ if [ "$JEKYLL_ENV" != "production" ]; then
     fi
 fi
 
-# Clone docs repo
-./scripts/build_clone_docs.sh "v20.0.0.11"
+# Determine which branch of docs-javadoc repo to clone
+BRANCH_NAME="prod"
```

Comment:
We should probably move this javadoc stuff into its own script since the other repos have their own, but for now let's just get it to work.